Consistent .js file names

fedgiac opened the pull request:
Rename all Javascript files sticking to snake case convention. Fixes #122.

josojo commented on .travis.yml:
```diff
@@ -14,4 +14,3 @@ script:
   - yarn lint
   - yarn pretty-check
   - yarn test
-  - npx mocha test/printingTools.js
```
How is this test called, once this line is removed?

bh2smith replied:
I don't understand the question here. Is it called or not? Why are we removing it?

fedgiac commented:
Sorry I didn't motivate the test change properly, probably it should have been its own micro-PR.
This line just uses Mocha directly for running the tests in the file instead of going through Truffle because it's much quicker. It comes from the time when we were including every single new test file manually into Travis.
The same test is now run while running `yarn test`. You can see [the build an old PR](https://travis-ci.com/github/gnosis/dex-liquidity-provision/builds/159668708) to see that a test is run twice (Ctrl+f `toErc20Units`). In this PR [it's still running, but only once](https://travis-ci.com/github/gnosis/dex-liquidity-provision/builds/159762873#L445).
I changed it because the Travis build of this PR was failing, since I renamed the file it was testing.
